README.md

```diff
@@ -82,9 +88,8 @@ python3 test/deploy.py build
 python3 test/deploy.py delete model-88HJ.json
 ```
 3.) Use Script to deploy a custom image:
-e.g. <image_name>: ceph/ceph:latest
 ```
-python3 test/deploy.py image <image_name>
+python3 test/deploy.py image ghcr.io/canonical/ceph:main
```

Comment:
This feels weird, as we're saying how to use a custom image, and then specify the image. I'd be tempted to reword this to use our image in one example, and mention that other images can be used as well, including a self-built one

Reply:
yeah, I will revert to the original <image_name> structure.